board/common/rootfs/usr/lib/finit/system/20-hw-wait.conf

```diff
@@ -0,0 +1,9 @@
+# Extend finit's default udevadm settle synchronization for situations
+# where device are very slow to probe (see #685)
+run nowarn if:udevd cgroup.init [S] <service/udevd/ready> log \
+	/usr/libexec/infix/hw-wait -- Probing hardware
+
+# Now that everything should be probed, do a final pass over the
+# uevent queue before starting syslogd and everything else
+run nowarn if:udevd cgroup.init :post [S] <service/udevd/ready> log \
+	udevadm settle -t 30 --
```

Comment:
I've taken as convention to start the second line with the runlevels, e.g.,

``` 
run nowarn if:udevd cgroup.init <service/udevd/ready> log \
	[S] /usr/libexec/infix/hw-wait -- Probing hardware
```

This makes it easier to spot when inspecting the configuration.  But it's just a very minor thing!

Reply:
TIL, thanks!